test/ng/directive/inputSpec.js

```diff
@@ -2948,7 +2948,8 @@ describe('input', function() {
       expect(inputElm[1].checked).toBe(true);
       expect(inputElm[2].checked).toBe(false);
 
-      browserTrigger(inputElm[2], 'click');
+      if (event === 'change') inputElm[2].checked = true;
```

Comment:
But then I guess you can't do much...
<sub>
Maybe we should document that for custom checkboxes/radios, where the change event is triggered manually, you need to ensure `.checked` has been set first. Or maybe it's obvious and doesn't need documentation..
</sub>